### PR TITLE
CYS: fix survey tracking

### PR DESCRIPTION
**Pull request by gigitux**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR is necessary to improve how we track the survey answers. Please for more details, check this private link: PCYsg-PWZ-p2.

This PR is a follow-up to #50196. For it to work correctly, we need to pass the `comments` fields.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Remove these code lines: https://github.com/woocommerce/woocommerce/blob/5d936bb547706a4fe1f2c24f7dda7a418d5fe814/packages/js/tracks/src/index.ts/#L52-L55 (or build a WooCommerce production build)
4. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
5. Click on `Design your homepage`.
6. Add a pattern.
8. Click save.
9. Click on share feedback.
10. Fill the survey
11. Open the network tools filter.
12. Submit the survey
13. Search `t.gif` in the network tools filter.
14. Ensure that `ces_feedback` event is tracked.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

CYS: Improve tracking survey

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx [hunk 1]
```diff
@@ -98,12 +98,12 @@ export const SurveyForm = ( {
...
 				action: 'customize_your_store_on_core_transitional_survey_complete',
-				rating,
+				score: rating,
 				choose_design_my_own_theme: isStreamlineChecked,
 				choose_dislike_themes: isDislikeThemesChecked,
 				choose_themes_not_match: isThemeNoMatchChecked,
 				choose_other: isOtherChecked,
-				feedback: feedbackText,
+				comments: feedbackText,
 				spill_beans: spillBeansText,
```
Comment (louwie17) [inline plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx:107]: This does mean that `spill_beans` won't be included in the FSOR.
May not a huge deal for now, but we could add an improvement to the FSOR to allow for `additional_comments` or something to that affect that would also allow us to import the above.
Or maybe we want to make this configurable through the JSON.

**Review by louwie17 — APPROVED** [review]
Tested well and changes are straight forward 👍 
Btw its also possible to test track data by using `localStorage.setItem( 'debug', 'wc-admin:*' );`
This adds a console log of the track, but in the `Verbose` level, so you did have to make sure those are also shown in your console.

File: plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx (hunk 1 above)
Comment (gigitux) [reply plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx:107]: What about the rest of the properties like `choose_design_my_own_theme`? Even if they do not show on FSOR, they are still available on our tracking infrastructure, right?

Comment (gigitux) [reply plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx:107]: I just checked and I can confirm they are still available! 🚀

Comment (louwie17) [reply plugins/woocommerce-admin/client/customize-store/transitional/survey-form/index.tsx:107]: Yes, everything is available in the tracking infrastructure

**Pull request merged**